tmt/cli/_root.py

```diff
@@ -1434,12 +1435,17 @@ def clean_images(context: Context, **kwargs: Any) -> None:
     #        cleaned, similarly to guests.
     assert context.obj.clean_logger is not None  # narrow type
 
+    workdir_root = Path(_workdir_root) if _workdir_root is not None else None
```

Comment:
@skycastlelily I am wondering if 

```suggestion
    workdir_root = Path(_workdir_root) if _workdir_root else None
```

would not be better. Feels like an empty string should also be evaluated to not set? WDYT?

Reply:
Ah, that line is from  Milos,yeah,I copied his suggestion from #2850  :sweat_smile: 
`workdir_root = Path(_workdir_root) if _workdir_root is not None else None
`
will  make tmt take the current dir ie, '.'  as workdir_root,
while, `workdir_root = Path(_workdir_root) if _workdir_root else None`
will make tmt take Path(os.environ['TMT_WORKDIR_ROOT']) or WORKDIR_ROOT .
And, with "workdir_root = Path(_workdir_root) if _workdir_root is not None else None"
users would run into an error: Run workdir 'run-001' must not be inside fmf root '/home/lnie/tmt'. if they run 
`tmt run --workdir-root ''
`
Which is the supposed behavior? @thrix  @happz 
Or, maybe, we should raise an error when users pass an empty string?:)

Reply:
I see, yeah, rather then error maybe?

Reply:
I believe this will become a moot point after https://github.com/teemtee/tmt/pull/3489

Reply:
yep, true, resolving